src/main/java/ch/njol/skript/effects/EffWorldBorderExpand.java

```diff
@@ -0,0 +1,92 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Expand/Shrink World Border")
+@Description({
+	"Expand or shrink the size of a world border.",
+	"Note: Using <code>by</code> adds/subtracts from the current size of the world border.",
+	"Using <code>to</code> sets to the specified size."
+})
+@Examples({
+	"expand world border of player by 100 in 5 seconds",
+	"shrink world border of world \"world\" to 100 in 10 seconds"
+})
+@Since("INSERT VERSION")
+public class EffWorldBorderExpand extends Effect {
+
+	static {
+		Skript.registerEffect(EffWorldBorderExpand.class,
+			"(expand|:shrink) (radius|:diameter) of %worldborders% (by|:to) %number% [over [[a ](time|period) of] %-timespan%]",
+			"(expand|:shrink) %worldborders%'s (radius|:diameter) (by|:to) %number% [over [[a ](time|period) of] %-timespan%]"
```

Comment:
I dont love the radius/diameter part being mandatory, it feels like the kind of thing that should still work when excluded.

Reply:
Which one should it default to? Bukkit uses diameter but I feel like radius is more intuitive

Reply:
I reckon radius too, but definitely safe to get some more opinions on this

Reply:
Which one does the minecraft command do? radius or diameter?

Reply:
It seems that minecraft also uses diameter so I will go with that as the default. Would it be better to have diameter as an option: `(expand|:shrink) [(radius|:diameter) of] %worldborders%...`, or should it be more like `(expand|:shrink) [:radius of] %worldborders%...`

Reply:
I'd like to keep it as an option so people can understand the code easier when reading it and not being expected to know default behavior